src/classes/forge/forge.ts

```diff
@@ -0,0 +1,84 @@
+import _ from "lodash";
+import { Document } from "mongoose";
+import { IForge } from "../../types/forge/forge";
+import { LocationEnum } from "../../enums/location.enum";
+
+export class Forge extends Document implements IForge {
+  public angelMash: IForge["angelMash"];
+  public bossId: IForge["bossId"];
+  public experience: IForge["experience"];
+  public furnitures: IForge["furnitures"];
+  public level: IForge["level"];
+  public location: IForge["location"];
+  public name: IForge["name"];
+
+  constructor() {
+    super();
+    this.angelMash = 100;
```

Comment:
Moving primitive values to declaration would be better

Reply:
Ok! I will move them.